client-app/i18n.ts

```diff
@@ -35,6 +43,12 @@ export function createI18n(locale: string, currency: string) {
         },
       },
     },
+    fallbackLocale: fallback?.locale,
+    messages: {
+      ...fallbackMessage,
+    },
+    fallbackWarn: false,
+    missingWarn: false,
```

Comment:
While `fallbackWarn` is discussable, `missingWarn` shouldn't be set to `false`, because in this case you will not receive warning about missed locale until you will see (if you will see) ugly localization key in UI